README.md

```diff
@@ -25,3 +25,31 @@ If the special pages work, congratulations! You successfully deployed PKC!
 * Visit https://github.com/FantasticoFox/VerifyPage for the chrome extension.
 
 Keep in mind that this software is in alpha-release stage. Please report bugs and issues back to us.
+
+## Repository Dependencies Github ##
```

Comment:
You have to explicitly write that these dependencies are automatically installed by `./pk`.